src/App.js

```diff
@@ -57,20 +66,21 @@ function App() {
   const user = useMemo(() => store.user, [store.user]);
 
   return (
-    <div style={{ height: "100vh", overflow: "scroll", ...theme }}>
+    <div style={{ height: "100vh", overflow: "hidden", ...theme }}>
```

Comment:
why overflow:hidden ?

Reply:
Incase of scroll, we get a scrollbar both for vertical and horizontal in dashboard page